utam-core/src/main/java/utam/core/selenium/element/DriverAdapter.java

```diff
@@ -221,6 +219,34 @@ public void forward() {
     this.driver.navigate().forward();
   }
 
+  @Override
+  public String getWindowHandle() { return this.driver.getWindowHandle(); }
+
+  @Override
+  public Set<String> getWindowHandles() { return this.driver.getWindowHandles(); }
+
+  @Override
+  public WebDriver.TargetLocator switchTo() { return this.driver.switchTo(); }
```

Comment:
None of the Driver interface API should ever return Selenium specific objects. Remove this method pls